misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -1290,6 +1290,7 @@ public IntersectionTypeDescriptorNode transform(IntersectionTypeDescriptorNode i
     @Override
     public ModuleVariableDeclarationNode transform(ModuleVariableDeclarationNode moduleVariableDeclarationNode) {
         MetadataNode metadata = formatNode(moduleVariableDeclarationNode.metadata().orElse(null), 0, 1);
+        Token visibilityQual = formatToken(moduleVariableDeclarationNode.visibilityQualifier().orElse(null), 1, 0);
```

Comment:
Do we need a formatting test for this? Can you create an issue to follow up on any changes?

Please check and create issues for doc generation and tooling support if required.